### ISSUE TITLE
frame name 

### ISSUE THREAD

**Issue by jiuyuehe** [tobiasahlin/SpinKit#110]
[http://tobiasahlin.com/spinkit/](http://tobiasahlin.com/spinkit/);

this site is very beautiful ,do you tell me the use the frame's name?

**Comment by tobiasahlin** [comment]
@jiuyuehe Thank you! 

> do you tell me the use the frame's name?

Could you rephrase this? What frame? And what do you want to know? :smile:

**Comment by PhiLhoSoft** [comment]
I don't know if that's the original meaning of the issue...

But indeed, when I look at the demo and see a nice animation, I would like to know which one to include.
For example, I see the fourth animation. First, I have to count the dots to have the number. It could be displayed somewhere. Second, I don't see the animation name. If I view the source, I see `sk-cubemove` inside it, but this name is nowhere to be found in the `scss/spinners` directory (looks like the demo is a bit out to date?).

So, in short, it would be nice to display "4-wandering-cubes" (in this example) somewhere on the page when selecting a demo.

Thanks.
